### ISSUE TITLE
the find window not always on top of the main window

### ISSUE THREAD

**Issue by czghzc** [arduino/Arduino#6408]
the find window not always on top of the main window

**Comment by facchinm** [comment]
As far as I can tell, the "find" window always appears at the center of the active IDE window. 
Are you experiencing something different? Can you provide a screenshot? Thanks!

**Comment by czghzc** [comment]
![2017-06-22_110300](https://user-images.githubusercontent.com/23645596/27422868-4d1f5c4e-5762-11e7-947c-15666d612d27.gif)

**Comment by facchinm** [comment]
Mmmmh, it looks like a problem with Windows' window manager  (and probably Java). I'll try investigating before updating to the next Java version.

**Comment by czghzc** [comment]
My system is Win7-32Bit.
My previous version of Java is 1.7.0_80-b15, there is such a problem.
Then I updated the Java version to 1.8.0_131-b11, the problem is still not resolved.
If you have no problem with the test, it may be my system.

**Comment by czghzc** [comment]
![2017-10-07_092746](https://user-images.githubusercontent.com/23645596/31303575-957ea1c8-ab42-11e7-9d30-7f0c71bdbf53.png)

I'm sorry, but now I understand your reply, it is ashamed.
I tested it, and now the find window is above all windows, but it finds another problem.
If there are other windows in the system, such as a browser, the find window will also be on top of the browser window, which will interfere with the operation of the browser window, and we hope that the find window is just above the Arduino main window.

**Comment by kenneth558** [comment]
Agreed that the find window drops behind main window.  Linux Mint 18, IDE 1.8.5, but has always been the case in IDE on Linux

**Comment by facchinm** [comment]
Would you mind testing https://github.com/arduino/Arduino/pull/6603 (or the latest beta)? The wrong behaviour should be gone for good.

**Comment by czghzc** [comment]
![pic2](https://user-images.githubusercontent.com/23645596/37202019-56b78d10-23c4-11e8-8436-fd2f649f2467.gif)
The beginning is good, there is no problem, but after clicking the window's title bar with a mouse is not normal, the search window goes to the back of the main window.

When programming in C++, setting the search window to the child window of the main window can achieve the desired effect. However, I will not use Java. I do not know if Java has the same operation method.
